tls/tls_internal.h

```diff
@@ -417,4 +417,46 @@ ttls_rnd(void *buf, int len)
 }
 #endif
 
+static inline char *
+tls_state_to_str(unsigned int state) {
+	switch (state) {
```

Comment:
Maybe we can convert this switch to table?

Reply:
Something like this? Is it faster then switch?

```
if (state > MAX) {
    return "Unknown";
} else {
    return table[state];
}
```

Reply:
Done. Is it good refactoring? Also, hstype_to_str is pretty big too, it can be converted in the same way.

Reply:
The switches are small and compiler and cpu are good in optimizing them and also the switches are only for error code paths.